GO: can not build the parser

```
# parser
./mysql_parser.go:13247:12: cannot use NewIntervalScheduleContext(p, localctx) (type *IntervalScheduleContext) as type IScheduleExpressionContext in assignment:
	*IntervalScheduleContext does not implement IScheduleExpressionContext (wrong type for GetStart method)
		have GetStart() ITimestampValueContext
		want GetStart() antlr.Token
./mysql_parser.go:13287:13: impossible type assertion:
	*IntervalScheduleContext does not implement IScheduleExpressionContext (wrong type for GetStart method)
		have GetStart() ITimestampValueContext
		want GetStart() antlr.Token
./mysql_parser.go:13299:14: impossible type assertion:
	*IntervalScheduleContext does not implement IScheduleExpressionContext (wrong type for GetStart method)
		have GetStart() ITimestampValueContext
		want GetStart() antlr.Token
./mysql_parser.go:13301:13: impossible type assertion:
	*IntervalScheduleContext does not implement IScheduleExpressionContext (wrong type for GetStart method)
		have GetStart() ITimestampValueContext
		want GetStart() antlr.Token
./mysql_parser.go:13323:13: impossible type assertion:
	*IntervalScheduleContext does not implement IScheduleExpressionContext (wrong type for GetStart method)
		have GetStart() ITimestampValueContext
		want GetStart() antlr.Token
./mysql_parser.go:13335:14: impossible type assertion:
	*IntervalScheduleContext does not implement IScheduleExpressionContext (wrong type for GetStart method)
		have GetStart() ITimestampValueContext
		want GetStart() antlr.Token
./mysql_parser.go:13337:13: impossible type assertion:
	*IntervalScheduleContext does not implement IScheduleExpressionContext (wrong type for GetStart method)
		have GetStart() ITimestampValueContext
		want GetStart() antlr.Token
```

--- a/mysql/MySqlParser.g4
+++ b/mysql/MySqlParser.g4
@@ -258,11 +258,11 @@ scheduleExpression
     : AT timestampValue intervalExpr*                               #preciseSchedule
     | EVERY (decimalLiteral | expression) intervalType
         (
-          STARTS start=timestampValue
+          STARTS startTimestamp=timestampValue
           (startIntervals+=intervalExpr)*
         )?
         (
-          ENDS end=timestampValue
+          ENDS endTimestamp=timestampValue
           (endIntervals+=intervalExpr)*
         )?                                                          #intervalSchedule
     ;

